CRM/Admin/Form/Setting/Search.php

```diff
@@ -80,4 +80,80 @@ public static function enableOptionOne(&$value) {
     return TRUE;
   }
 
+  /**
+   * Pseudoconstant callback for autocomplete_displays setting.
+   *
+   * @return array
+   */
+  public static function getAutocompleteDisplays(): array {
```

Comment:
I don't love these BAO-ish functions being on this form - but I assume there is precedent / no obvious other place

Reply:
I don't love it either but it would be an easy refactor to move them to a better place if we think of one.